src/main/config/node-7443.cfg

```diff
@@ -1,6 +1,4 @@
 # The most basic node configuration
-URL = "@{emissary.node.scheme}://@{emissary.node.name}:@{emissary.node.port}"
-
 PLACE != "*"
```

Comment:
May need to rethink how we are wiring up the place configs...```PLACE != "*"``` will not work anymore. May have to use something like:
```PLACE = "FilePickUpClient|emissary.pickup.file.FilePickUpClient"```

Reply:
Going with 
```PLACE = "FilePickUpClient:emissary.pickup.file.FilePickUpClient"```